Automattic/newspack-plugin
# Adds Newspack Network & Multibranded Site

### All Submissions:






### Changes proposed in this Pull Request:



Adds the [Newspack Network](https://github.com/Automattic/newspack-network/) & [Multibranded Site](https://github.com/Automattic/newspack-multibranded-site/) plugins as managed plugins under Newspack Plugin. They're set to `quiet`, which means they will not be listed under Newspack Plugin until after they're installed.

Should not be merged until after Newspack Network has a GitHub release.

### How to test the changes in this Pull Request:

1. Visit `/wp-admin/plugins.php`
2. Confirm that Manager & Multibranded are not listed.
3. Manually install the plugins by uploading a ZIP.
4. Confirm that Manager & Multibranded are listed separately, not hierarchically under "Newspack".
5. Delete the two plugins.
6. Switch to this branch.
7. Visit `/wp-admin/plugins.php`
8. Confirm that Manager & Multibranded are still not listed.
9. Manually install the plugins by uploading a ZIP (or via symlink on the Atomic platform).
10. Confirm that Manager & Multibranded are listed hierarchically under "Newspack".

### Other information:
